### PR TITLE
Ability to nominate

### LINKED ISSUE #730: Ability to nominate

### PR DESCRIPTION
**Pull request by ajsantander**
Fix #730

### CODE REVIEW

File: packages/core-modules/contracts/interfaces/IElectionModule.sol [hunk 1]
```diff
@@ -37,4 +37,8 @@ interface IElectionModule {
     function getVotingPeriodStartDate() external view returns (uint64);
 
     function isEpochEvaluated() external view returns (bool);
+
+    function isNominated(address candidate) external view returns (bool);
+
+    function getNominees() external view returns (address[] memory);
```
Comment (leomassazza) [inline packages/core-modules/contracts/interfaces/IElectionModule.sol:43]: this could be long and may require pagination. If not part of this PR, please add a ticket to investigate it

**Review by leomassazza — APPROVED** [review]
Approved. Only one comment to check

File: packages/core-modules/contracts/interfaces/IElectionModule.sol (hunk 1 above)
Comment (ajsantander) [reply packages/core-modules/contracts/interfaces/IElectionModule.sol:43]: Let's ignore pagination in all view functions for now.
This means that we need only consider pagination during an election evaluation.

**Review by mjlescano — APPROVED** [review]
